Remotion/Web/Development.WebTesting.UnitTests/Configuration/TestInfrastructureConfigurationTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Coypu;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests.Configuration
+{
+  [TestFixture]
+  public class TestInfrastructureConfigurationTest
+  {
+    private readonly string _testRequestErrorDetectionStrategyAssemblyQualifiedName = typeof (TestRequestErrorDetectionStrategy).AssemblyQualifiedName;
+
+    private readonly MethodInfo _webTestConfigurationFactoryPropertySetter = typeof (WebTestConfigurationSection)

```

Comment:
kann man static machen.
Bzw. haben wir in dem Projekt eine dependency auf Remotion.Development? Dort gibt es ein PrivateInvoke Utility drinnen.

Reply:
`PrivateInvoke` scheint auf den ersten Blick nicht zwischen `object this[string]` und `object this[ConfigurationProperty]` unterscheiden zu können. Übersehe ich etwas?

Reply:
Are you suuuuure?

Reply:
`System.Reflection.AmbiguousMatchException : Ambiguous match found.`.

Reply:
Not for me:
`PrivateInvoke.InvokeNonPublicMethod (config, "get_Item", "downloadStartedTimeout")`

Reply:
My bad 🙈